e2e/support/config.js

```diff
@@ -23,7 +23,7 @@ const isQaDatabase = process.env["QA_DB_ENABLED"];
 const sourceVersion = process.env["CROSS_VERSION_SOURCE"];
 const targetVersion = process.env["CROSS_VERSION_TARGET"];
 
-const runWithReplay = process.env["REPLAYIO_ENABLED"];
+const runWithReplay = process.env["CYPRESS_REPLAYIO_ENABLED"];
```

Comment:
I could've maybe used `Cypress.env("REPLAYIO_ENABLED)` here as well, but:
1. didn't want to risk it
2. wanted to keep reading system environment variables in this file uniform

Reply:
what kind of risk? if there is some env variable which shows explicitly that REPLAYIO is enabled, I'd just reuse it.

Reply:
Risk that it doesn't work.

> if there is some env variable which shows explicitly that REPLAYIO is enabled, I'd just reuse it.

There is. `CYPRESS_REPLAYIO_ENABLED` :)

Reply:
@uladzimirdev to clarify - even the previously used `REPLAYIO_ENABLED` didn't mean anything globally. It was just a name we picked to limit the scope of Replay to certain tests only. Replay's integration with Cypress is still WIP so we don't want it to interfere with our regular runs.